### PR TITLE
Add unit tests with industry standard tools

### LINKED ISSUE #4: cover the codebase with unit tests
make sure to choose the best framework available on the market. cc @Marcel-G

### PR DESCRIPTION
**Pull request by Marcel-G**
Resolves https://github.com/mattp91/helloWorld/issues/4

Assigning to @AlejandroE & @agudulin, since they're familiar with the framework

### CODE REVIEW

File: tests/main.spec.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "dog should be not too far from the freezer": function () {
+      var security = fs.readFileSync("./security.md", { encoding: "utf8" });
+      buster.expect(security).toContain("🐕");
+      buster.expect(security).toContain("🥶");
+
+      buster
+        .expect(Math.abs(security.indexOf("🐕") - security.indexOf("🥶")))
+        .toBeLessThan(5);
+    },
+    "puppies are cool to": function () {
```
Comment (mattp91) [inline tests/main.spec.js:15]:
```suggestion
    "puppies are cool too": function () {
```

**Review by mattp91 — COMMENTED** [review]
⽝

**Review by agudulin — APPROVED** [review]
🥝 as 🚁 !

File: package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "url": "git+https://github.com/mattp91/helloWorld.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/mattp91/helloWorld/issues"
+  },
+  "homepage": "https://github.com/mattp91/helloWorld#readme",
+  "devDependencies": {
+    "buster": "^0.7.18"
```
Comment (AlejandroE) [inline package.json:20]: ❤️‍🔥

**Review by AlejandroE — APPROVED** [review]
🚌 🦖

**Review by mattp91 — APPROVED** [review]
Thank you, I find this testing is just the kind of modern futurist testing an avant garde project like this requires.

Cheers Buster 🥂

**Pull request merged**